randovania/gui/preset_settings/generation_tab.py

```diff
@@ -1,10 +1,14 @@
 import dataclasses
 from typing import Iterable
 
+from PySide6 import QtWidgets, QtCore
```

Comment:
you actually shouldn't need to edit this file at all. you can create a subclass of this for Dread - check out what prime1 does for an example of adding game-specific widgets to the generation tab

Reply:
I've made it game-specific now.